### PR TITLE
Update en.lua of qb-towjob

### PR DESCRIPTION
**Pull request by 12LetterMeme**
updating some of the grammar used in towjob

### CODE REVIEW

**Comment by MatthewRorke** [comment]
Can you please fix the capitalisations

File: locales/en.lua [hunk 1]
```diff
@@ -1,11 +1,11 @@
 local Translations = {
     error = {
-        finish_work = "First Finish Your Work",
+        finish_work = "Finish All Of Your Work First",
         vehicle_not_correct = "This Is Not The Right Vehicle",
```
Comment (MatthewRorke) [inline locales/en.lua:4]:
```suggestion
        vehicle_not_correct = "This is not the right Vehicle",
```

File: locales/en.lua [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
-        failed = "Failed",
-        not_towing_vehicle = "You Must Have Been In A Towing Vehicle First",
-        too_far_away = 'You are too far away',
-        no_work_done = "You have not done any work yet.",
+        failed = "You Have Failed",
```
Comment (MatthewRorke) [inline locales/en.lua:5]:
```suggestion
        failed = "You have failed",
```

File: locales/en.lua [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
+        not_towing_vehicle = "You Must Be In Your Towing Vehicle",
+        too_far_away = "You Are Too Far Away",
```
Comment (MatthewRorke) [inline locales/en.lua:7]:
```suggestion
        too_far_away = "You are too far away",
```

File: locales/en.lua [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
+        no_work_done = "You Have Not Done Any Work Yet",
```
Comment (MatthewRorke) [inline locales/en.lua:8]:
```suggestion
        no_work_done = "You have not done any work yet",
```

**Review by MatthewRorke — CHANGES_REQUESTED** [review]
Hey, sorry I was not specific enough. That's my bad.
I suggested the changes to fix the capitalisations (we shouldn't be capitalising every single word). 

You should be able to just accept the changes from my review and then we can proceed with merging. Thanks for contributing.

File: locales/en.lua [hunk 5]
```diff
@@ -1,11 +1,11 @@
...
         no_deposit = "$%{value} Deposit Required",
```
[the remark this answers is not in the document]
Comment (12LetterMeme) [reply locales/en.lua:9]: I was thinking the same thing initially, but just decided to go with what was originally there which was everything was capitalized.. thought there was a reason for it lmao

**Comment by 12LetterMeme** [comment]
sorry I made this painful haha

**Comment by MatthewRorke** [comment]
Not painful at all. People get used to writing in PascalCase and end up writing that way in English, lmao.

Please can you just update your branch to the latest version as we did some merges yesterday and then I will get yours merged in.

**Pull request merged**